monai/losses/dice.py

```diff
@@ -835,6 +835,7 @@ def __init__(
         gamma: float = 2.0,
         focal_weight: Sequence[float] | float | int | torch.Tensor | None = None,
         weight: Sequence[float] | float | int | torch.Tensor | None = None,
+        alpha: float | None = None,
```

Comment:
To avoid breaking any existing code, new parameters should appear at the end of the list. I know it makes sense for the lambda parameters to be last but this avoids problems if anyone called the constructor with all positional arguments.